firmware/application/freqman.cpp

```diff
@@ -269,50 +285,52 @@ if (MY_DEBUG) 	{
 		if (read_size.value() != 256) {break;	}																				// End of file
 			
 		// Restart at beginning of last incomplete line		
-		
 	if (line_start != file_data )    {	file_position -= (file_data + 256 - line_start); }
 	}
-		
 	return true;
 }
 
 bool save_freqman_file(std::string& file_stem, freqman_db &db) {
 	File freqman_file;
 	std::string item_string;
+	std::string description;
 	rf::Frequency frequency_a, frequency_b;
 	
+//	DEBUG (3,"IN SAVE:" + file_stem );   
+	
 	if (!create_freqman_file(file_stem, freqman_file))
 		return false;
 	
 	for (size_t n = 0; n < db.size(); n++) {
 		auto& entry = db[n];
 
 		frequency_a = entry.frequency_a;
-	
-		
+//		DEBUG (3,"IN SAVE n:" + to_string_dec_uint(n) + " typ; " +to_string_dec_uint(entry.type) );
 	switch (entry.type) 	{
-		case 0:		item_string = "f=" + to_string_dec_uint(frequency_a / 1000) + to_string_dec_uint(frequency_a % 1000UL, 3, '0');	 	break ;			//SINGLE
+		case 0:		item_string = "f=" + to_string_dec_uint(frequency_a / 1000) + to_string_dec_uint(frequency_a % 1000UL, 3, '0');
+					if (entry.description.size()) {item_string += ",d=" + entry.description;}	 	
+					break ;								//SINGLE
 		case 1: 	frequency_b = entry.frequency_b;
 					item_string = "a=" + to_string_dec_uint(frequency_a / 1000) + to_string_dec_uint(frequency_a % 1000UL, 3, '0');
-					item_string += ",b=" + to_string_dec_uint(frequency_b / 1000) + to_string_dec_uint(frequency_b % 1000UL, 3, '0');  	break ;												// RANGE
-		case 2:    	break ;																									// ERROR																					
-		default:		;																									// error				
+					item_string += ",b=" + to_string_dec_uint(frequency_b / 1000) + to_string_dec_uint(frequency_b % 1000UL, 3, '0');
+					if (entry.description.size()) {item_string += ",d=" + entry.description;}
+					break ;									//RANGE			
+		case 2:    																										// ERROR NO BREAK
+		case 3:		item_string = description;	break;																	// COMMENT
+		default:	item_string = "BIG ERROR";																				// error				
 							} 
-	
-
-		if (entry.description.size())
-			item_string += ",d=" + entry.description;
-		freqman_file.write_line(item_string);
+//		DEBUG (3,"OUT SAVE<" + item_string + "<");
+		freqman_file.write_line_n(item_string);
```

Comment:
write_line_n ?
This throws an error on my compiler.
@alain00091 Do you try to compile your code against @eried branch, before submitting the PRs ?